.github/workflows/glass-unit-tests.yml

```diff
@@ -0,0 +1,24 @@
+name: ci
```

Comment:
Lets give this a more descriptive name so that on the "Checks" tab it's clear what this is.

Maybe "NPM Tests"?

Reply:
@jhesketh npm is the package manager for node.js, the actual test command under the hood is _jest_ which is npm package. In other words npm is used not only for running tests but for building and packaging. Why ci / glass is not descriptive enough?

Reply:
@jhesketh maybe we can merge it as is, and in upcoming PR you can play with the name you like more as for Checks?

Reply:
I'm not particularly invested in this discussion, but, maybe, then naming it "glass unit tests" would suffice?

Reply:
Right, my example given is clearly poor, but my point was to have something other than CI because we have other systems also doing tests so we should differentiate them. "Glass unit tests" makes sense to me.